concepts/05 UI Components/TreeView/00 Getting Started with TreeView/50 Select Records.md

```diff
@@ -0,0 +1,282 @@
+To enable node selection, you can use the following properties:
+
+* [selectByClick](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#selectByClick)           
+Enables selection by click.
+
+* [showCheckBoxesMode](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#showCheckBoxesMode)           
+Adds checkboxes. Set this property to *"normal"* or *"selectAll"*. The latter additionally displays the "Select All" checkbox at the top of the TreeView.
```

Comment:
From what I understand, the checkboxes are disabled but you can set the mentioned property to `normal` or `selectAll` to enable them. I understand that you may want to start the line with "Adds checkboxes" to maintain the pattern introduced in the previous list item. However, this variant looks more natural to me:

```suggestion
Set this property to *"normal"* or *"selectAll"* to add checkboxes. The latter option additionally displays the "Select All" checkbox at the top of the TreeView.
```